app/src/main/java/org/simple/clinic/summary/bloodsugar/BloodSugarSummaryViewModel.kt

```diff
@@ -1,16 +1,20 @@
 package org.simple.clinic.summary.bloodsugar
 
+import org.simple.clinic.bloodsugar.BloodSugarMeasurement
 import java.util.UUID
 
-data class BloodSugarSummaryViewModel(val patientUuid: UUID) {
+data class BloodSugarSummaryViewModel(
+    val patientUuid: UUID,
+    val measurements: List<BloodSugarMeasurement>
```

Comment:
When I see an empty list here, it could possibly be for two reasons:

- Either the data was never queried
- Or the data was queried, and was found to be empty

Which means that treating these two cases separately is not possible when representing in this manner. We've been using nullable lists in models for this reason. Can we make this a nullable list instead?

Reply:
I would want to do this when I write the `UiRenderer` if that makes sense.
I will have to handle the `No blood sugars` state.